src/main/java/net/trapezokomos/dashboard/controller/UserController.java

```diff
@@ -0,0 +1,85 @@
+package net.trapezokomos.dashboard.controller;
+
+import jakarta.validation.Valid;
+import net.trapezokomos.dashboard.resources.UserResource;
+import net.trapezokomos.dashboard.service.UserService;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/user")
```

Comment:
lets move the api to "api/v1/user" to be able to easily serve html on root or not and have api versioning